shared/fs/footer/downloads.js

```diff
@@ -0,0 +1,68 @@
+// @flow
+import * as React from 'react'
+import {globalStyles, globalMargins, globalColors, styleSheetCreate, isMobile} from '../../styles'
+import {Box2, Box, Icon, ClickableBox} from '../../common-adapters'
+import Download from './download-container'
+
+export type DownloadsProps = {
+  downloadKeys: Array<string>,
+  openDownloadFolder?: () => void,
+}
+
+const maxNumCards = isMobile ? 1 : 3
```

Comment:
Just limit the cards in the connector

Reply:
The component also decides whether to show the "..." button. If we do that container, we'd need an additional prop for whether to show the "..." button, so component seemed like a better place to me. Would you mind explaining why it's better to put it in container here?

Reply:
i think its better if the components are dumb and the containers are smart. especially if the component is split into native/desktop
if it was me (you don't have to change this, but)
I'd make
```
cards: Array<whatever>
thereAreMoreCard: boolean 
```
or something, that encodes all you need

Reply:
OK; will do. Thanks!